tests/antithesis/Makefile

```diff
@@ -13,16 +13,20 @@ antithesis-build-etcd-image:
 	docker build --build-arg GO_VERSION=$(shell cat $(REPOSITORY_ROOT)/.go-version) --build-arg REF=${REF} $(REPOSITORY_ROOT)/tests/antithesis/server/ -t etcd-server:latest
 
 .PHONY: antithesis-build-etcd-image-release-3.4
-antithesis-build-etcd-image-release-3.4: set-version-3.4 antithesis-build-etcd-image
+antithesis-build-etcd-image-release-3.4: REF=release-3.4
+antithesis-build-etcd-image-release-3.4: antithesis-build-etcd-image
 
 .PHONY: antithesis-build-etcd-image-release-3.5
-antithesis-build-etcd-image-release-3.5: set-version-3.5 antithesis-build-etcd-image
+antithesis-build-etcd-image-release-3.5: REF=release-3.5
+antithesis-build-etcd-image-release-3.5: antithesis-build-etcd-image
 
 .PHONY: antithesis-build-etcd-image-release-3.6
-antithesis-build-etcd-image-release-3.6: set-version-3.6 antithesis-build-etcd-image
+antithesis-build-etcd-image-release-3.6: REF=release-3.6
+antithesis-build-etcd-image-release-3.6: antithesis-build-etcd-image
 
 .PHONY: antithesis-build-etcd-image-main
-antithesis-build-etcd-image-main: set-version-main antithesis-build-etcd-image
+antithesis-build-etcd-image-main: REF=main
```

Comment:
`REF` is set in line 5, so it has a default value set to `main`. So, setting it here may be unnecessary.

Reply:
This is meant to maintain same name pattern for branches like `release-3.5` and `main`. Current default is `main` but we might change it in the future.